File: sdkwork-app-sdk-typescript/src/types/partner-create-form.ts
/** 创建合作伙伴表单 */
export interface PartnerCreateForm {
  /** 名称 */
  name: string;
  /** 联系人 */
  contactName?: string;
  /** 联系电话 */
  contactPhone?: string;
  /** 联系邮箱 */
  contactEmail?: string;
  /** 等级 */
  level?: string;
  /** 上级ID */
  parentId?: number;
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/partner_create_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PartnerCreateForm:
    """创建合作伙伴表单"""
    name: str
    contact_name: str = None
    contact_phone: str = None
    contact_email: str = None
    level: str = None
    parent_id: int = None
